src/applications/resources-and-support/components/ResourcesAndSupportSearchApp.jsx

```diff
@@ -73,10 +73,10 @@ const ResourcesAndSupportSearchApp = () => {
 
   if (results.length > 0) {
     paginationSummary = (
-      <>
+      <h2 className="vads-u-font-size--base">
```

Comment:
I think if you do this you will have an `h2` inside a `p` tag because of:
```jsx
            <p
              className="vads-u-padding-x--1 large-screen:vads-u-padding-x--0"
              id="pagination-summary"
            >
              {paginationSummary}
            </p>
```

Reply:
i'm an idiot and can't read lol

Reply:
Also if you wanted we could abstract this if logic for `paginationSummary` into a helper method called `const paginationSummary = derivePaginationSummary()` so that it's a bit easier to read 🙂